src/app/ubs/ubs-user/ubs-user-messages/ubs-user-messages.component.html

```diff
@@ -27,13 +27,13 @@
                     totalItems: count
                   }
           "
-          (click)="setRead(notification.id, notification.read)"
+          (click)="setRead(notification.orderId, notification.read)"
```

Comment:
Change please from **orderId** to **id**.

Reply:
Done! I hope it will be final stage our task :)